src/components/Header/Header.styled.jsx

```diff
@@ -0,0 +1,289 @@
+    ${elementMenu}
+    border-right: 1px solid #858598;
+    color: rgb(133, 133, 152);
+`;
+
+export const MenuContact = styled.div` 
+    ${elementMenu}
+`;
+
+// ----------- styled for - HeaderDesktopAuth ----------
+export const PointsBalance = styled.div`
+    max-width: 85px;                         
+    color: #A6ABB9;
+    display: flex;
+    ${fontRegular}; 
+    @media (min-width: 1000px){
+        margin-left:130px;
+    }
+    @media (max-width: 1000px){
+        margin-left:40px;
+    }
+    @media (max-width: 330px){
+        margin-left:20px;
+    }
+`;
+
+export const NumberBalance = styled.div`
+    margin-left: 8px;
+    display: flex;
+    font-family: 'Montserrat-700';
+    font-size: 14px;
+    line-height: 17px;
+    align-items: center;
+    letter-spacing: 0.04em;
+    color: #000000;
+`;
+
+export const MenuSection = styled.div`
+    ${elementMenu}
+    border-right: 1px solid #858598;
+`;
+
+export const MenuInitiaLetterCircle = styled.div`
+    background: #2196F3;
+    width: 26px;
+    height: 26px;
+    border-radius: 50%;
+    margin-left: 24px;
+    display: grid;
+    align-items: end;
+    @media (max-width: 531px){
+        margin-left:20px;
+    }
+`;
+
+export const MenuInitiaLetter = styled.div`
+    width: 26px;
+    height: 24px;
+    font-family: 'Montserrat-700';
+    font-size: 17px;
+    line-height: 22px;
+    display: -ms-flexbox;
+    letter-spacing: 0.04em;
+    color: #FFFFFF;
+    display: grid;
+    align-items: end;
+    justify-content: center;
+`;
+
+export const MenuName = styled.div`
+    margin-left: 8px;
+    padding-right: 16px;
+    height: 19px;
+    display: grid;
+    align-items: center;
+    font-family: 'Montserrat-700';
+    font-size: 12px;
+    line-height: 15px;
+    letter-spacing: 0.04em;
+    border-right: 1px solid #858598;
+    color: #858598;
+    @media (max-width: 531px){
+        border-right: 1px solid white;
+        color: white;
+    }
+`;
+
+// ----------- styled for - HeaderMobile ----------
+export const MobileHamburgerLine = styled.div`
+    border-bottom: 1px solid #858598;
+`;
+
+export const MobileHamburgerContainer = styled.nav`
+    position: relative;
+    width: 22px;
+    height: 14px;
+    border-top: 1px solid #858598;
+    display: flex;
+    flex-direction: column;
+    border-bottom: 1px solid #858598;
+    justify-content: center;
+    cursor: pointer;
+    &:hover {
+        width: 23px;
+        height: 15px;
+        border-top: 1px solid black;
+        border-bottom: 1px solid black;
+        box-shadow:         
+        rgb(0 0 0 / 19%) 1px 3px 20px 0px, 
+        inset -10px -10px 15px rgb(255 255 255 / 80%), 
+        inset 10px 10px 15px rgb(70 70 70 / 12%);
+        ${MobileHamburgerLine}{
+            border-bottom: 1px solid black;
+        }
+    }
+    @media (max-width: 300px){
+        margin-left:20px;
+    }
+`;
+
+// -------------------------------------------------
+export const MobileHamburgerbackground = styled.nav`
```

Comment:
commented out code should be deleted

Reply:
Yes, exactly, I will remove it from the code)